mimic/text_generator.py

```diff
@@ -1,5 +1,7 @@
 """Core text generator module."""
-from mimic.model.model import Model
+# from mimic.model.model import Model
```

Comment:
If we don't need these we can just get rid of them.

Reply:
Turns out we don't need it because Python is dynamically typed (courtesy of @Annnnnieee). Left it in for clarity.